umap/views.py

```diff
@@ -780,6 +784,9 @@ def is_starred(self):
             return False
         return Star.objects.filter(by=user, map=self.object).exists()
 
+    def stars(self):
+        return Star.objects.filter(map=self.object).count()
```

Comment:
I wonder if we should store this count in the map itself, updating it at each star/unstar, instead of requesting this at each map view.

Reply:
I'm not sure it's necessary, a SQL count is relatively light.